src/super_gradients/common/decorators/code_save_decorator.py

```diff
@@ -0,0 +1,29 @@
+import inspect
+
+import os
+
+saved_codes = {}
+
+
```

Comment:
Can we add PEP8 docstrings to the below methods ?
Also, would be nice to see inside their docs an example usage with input/output as it might be a bit vague.